src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/nowhitespaceafter/InputNoWhitespaceAfterTestDefault.java

```diff
@@ -308,3 +308,30 @@ Object foo() {
     int someStuff8
     []; // violation
 }
+
+class QualifiedAssignment {
```

Comment:
Please create new test input so that we do not add to work required at https://github.com/checkstyle/checkstyle/issues/11163

Reply:
done